docs/user-guide/performance/parameters.md

```diff
@@ -15,11 +15,19 @@ helm upgrade --install \
 --set http.url=https://httpbin.org/get
 ```
 
-All the parameters of the performance test (including of all included tasks) are optional unless otherwise documented.
+All the parameters of the performance test (including of all included tasks) are optional unless otherwise documented. The task specific parameters are documented in the task documentation.
```

Comment:
```suggestion
All the parameters of the performance test (including of all included tasks) are optional unless otherwise documented. The task-specific parameters are documented in the task documentation.
```